services/dedup/mirror.go

```diff
@@ -13,14 +15,12 @@ import (
 	obskit "github.com/rudderlabs/rudder-observability-kit/go/labels"
 )
 
-const (
-	defaultMaxRoutines = 3000
-)
-
-type mirror struct {
-	Dedup
-	mirror Dedup
+const defaultMaxRoutines = 3000
 
+type MirrorDB struct {
```

Comment:
doesn't need to be exported
```suggestion
type mirrorDB struct {
```